target/linux/ath79/dts/qca9533_mikrotik_routerboard-wap-2nd.dts

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca9533_mikrotik_routerboard-16m.dtsi"
+
+/ {
+	compatible = "mikrotik,routerboard-wap-2nd", "qca,qca9533";
+	model = "MikroTik RouterBOARD wAP-2nD (wAP)";
+
+	aliases {
+		led-boot = &led_user;
+		led-failsafe = &led_user;
+		led-running = &led_user;
+		led-upgrade = &led_user;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		pinctrl-names = "default";
+		pinctrl-0 = <&led_lan_pin>;
+
+		lan {
+			label = "green:lan";
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan {
+			label = "green:wlan";
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		led_user: user {
+			label = "green:user";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+
+		};
+	};
+};
+
+&eth0 {
+	status = "okay";
+
+	phy-handle = <&swphy0>;
+
+	gmac-config {
+		device = <&gmac>;
+		switch-phy-swap = <1>;
+	};
+};
+
+&eth1 {
+	compatible = "syscon", "simple-mfd";
+};
+
+
+&pinmux {
+	led_lan_pin: pinmux_led_lan_pin {
+		pinctrl-single,bits = <0x4 0x0 0xff>;
+	};
+};
```

Comment:
Is this necessary? Looking at the [ar71xx code](https://github.com/openwrt/openwrt/blob/698cdf02020aa9855ab1000d6a359f816c76280e/target/linux/ar71xx/files/arch/mips/ath79/mach-rbspi.c#L1027), we didn't touch the pinmux on this device.

Reply:
Without this, the lan led does not work and cannot be controlled. 

Inspiration to use this pinmux is from [here](https://github.com/openwrt/openwrt/blob/ed364cd4b03b289a7bb4b32c350c46eddeb6b6b9/target/linux/ath79/dts/qca9533_mikrotik_routerboard-mapl-2nd.dts#L60), similar device, leds on the same gpio outputs.

Reply:
OK. I'm the author of the mapl commit you mention, so I'm familiar with this code :)

I have a wap-2nd lying around somewhere, I will test your PR next week.

Meanwhile the alphabetical order is still wrong ("wap-2" comes before "wap-g"), and you might also want to add a note in the commit message about the network setup (single port device, switch bypassed) as was done in eb38af788